src/assets/scripts/client/InputController.js

```diff
@@ -221,11 +222,12 @@ export default class InputController {
      * @private
      */
     _addMeasurePoint(event, shouldReplaceLastPoint = false) {
-        const currentMousePosition = CanvasStageModel.translateMousePositionToCanvasPosition(
-            event.pageX, event.pageY
-        );
-        const { x, y } = currentMousePosition;
-        let modelToUse = this._translatePointToKilometers(x, y);
+        // const currentMousePosition = CanvasStageModel.translateMousePositionToCanvasPosition(
+        //     event.pageX, event.pageY
+        // );
+        // const { x, y } = currentMousePosition;
+        // let modelToUse = this._translatePointToKilometers(x, y);
+        let modelToUse = this._getMouseRelativePositionFromEvent(event);
```

Comment:
is there a more specific name that could be used here?  is this a `position` of some sort?  Also, what are we _using_ this for?  

perhaps something like `positionToStartMeasurement` or something similar

Reply:
more specifically, it's a relative position. just updated the method name on this line in a find-and-replace really, but a good callout and it now is clearer what it does 👍